assets/css/styles.css

```diff
@@ -107,6 +107,259 @@ a {
 .icons:hover {
   transform: translateY(10px);
   transition: 0.5s;
+}
+
+#work-section {
+  display: block;
+  margin-top: 46px;
+  padding: 1em 1em;
+}
```

Comment:
- [ ] As mentioned in the requirements, you should use display: grid in this main container of the project cards, then in the individual card, you can use flex. kindly consider fixing it.
- In order to lay out the elements in "Works" section you should use CSS Grid and Flexbox combined (it is mandatory that you use both). Use CSS Grid in the main container that contains all of the cards and use Flexbox to lay out an individual card.